### PR TITLE
[METEOR 3]: Removing Fibers from Mocha

### PR DESCRIPTION
**Pull request by Grubba27**
As explained here: https://github.com/meteor/meteor/pull/12515 we have a few changes that needed to be done in mocha in order to work with Meteor 3
Relates to this one aswell https://github.com/Meteor-Community-Packages/meteor-mocha-core/pull/9

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
I will prepare a special release so this can be tested.

**Comment by Grubba27** [comment]
This is currently running our CI in the core. We have published a version to test it.

File: package/package.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ Package.describe({
   summary: 'Run Meteor package or app tests with Mocha',
   git: 'https://github.com/meteortesting/meteor-mocha.git',
   documentation: '../README.md',
-  version: '2.1.0',
+  version: '3.0.3-alpha300.11',
```
Comment (StorytellerCZ) [inline package/package.js:6]: The version you have here makes no sense. I get the major version bump, but where do the 3 patch versions come from?
Just keep it like `3.0.0-alpha.11` and increment the alpha number.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: package/package.js (hunk 1 above)
Comment (Grubba27) [reply package/package.js:6]: I agree! Before merging, I will correct this one. I did not realize that this was wrong. Thanks for the ping

**Review by jankapunkt — APPROVED** [review]

**Pull request merged**